docs-ref-services/latest/eventgrid.md

```diff
@@ -31,32 +31,24 @@ To manage your Azure Event Grid resources like Event Grid instances, topics, and
 
 Create events, authenticate, and post to topics using the below packages
 
-### v2 of @azure/eventgrid
+### Latest Packages
 
-To create events, authenticate, and post to topics, you would use the `@azure/eventgrid` package.
+To create events, authenticate, and post to topics, we recommend you use the latest version of the `@azure/eventgrid` package.
 
 | NPM Package                                                        | Reference                                                                                            |   
 | ------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------- | 
-| [@azure/eventgrid v2](https://npmjs.com/package/@azure/eventgrid) | [API Reference for @azure/eventgrid v2](https://docs.microsoft.com/javascript/api/@azure/eventgrid) | 
+| [@azure/eventgrid](https://www.npmjs.com/package/@azure/eventgrid) | [API Reference for @azure/eventgrid v3](https://docs.microsoft.com/javascript/api/@azure/eventgrid/) |
```

Comment:
```suggestion
| [@azure/eventgrid](https://www.npmjs.com/package/@azure/eventgrid) | [API Reference for @azure/eventgrid](https://docs.microsoft.com/javascript/api/@azure/eventgrid/) |
```

Reply:
Can we add a column for Samples and point it to https://docs.microsoft.com/en-us/samples/azure/azure-sdk-for-js/eventgrid-javascript/?

Reply:
Done